storagenode/pieces/readwrite.go

```diff
@@ -113,14 +302,17 @@ func (r *Reader) Seek(offset int64, whence int) (int64, error) {
 
 // ReadAt reads data at the specified offset
```

Comment:
We need to update comment that the offset here is relative to the underlying blob, not including the piece header size.